src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -271,7 +302,8 @@ class ErgoWalletActor(settings: ErgoSettings,
       secretStorageOpt.foreach(_.lock())
 
     case GetWalletStatus =>
-      val status = WalletStatus(secretIsSet, walletVars.proverOpt.isDefined, changeAddress)
+      val lastScannedHeight = registry.fetchDigest().height
```

Comment:
This seems like a better name instead of `fullHeight`. But maybe I just don't understand the difference, in which case the code need readability improvements.

Reply:
externalized registry.fetchDigest().height calls into walletHeight(), made a comment